phy/stats/ccg.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+
+"""Cross-correlograms."""
+
+#------------------------------------------------------------------------------
+# Imports
+#------------------------------------------------------------------------------
+
+import numpy as np
+
+from ..ext import six
+from ..utils.array import _index_of, _unique
+
+
+#------------------------------------------------------------------------------
+# Cross-correlograms
+#------------------------------------------------------------------------------
+
+def _increment(arr, indices):
+    """Increment some indices in a 1D vector of non-negative integers.
```

Comment:
Yes I think that's something we can do in the near future. There would be no overhead with such an `if`, here the overhead would probably be due to an array operation like `max()` on a big array.